.github/issue_template.md

```diff
@@ -0,0 +1,18 @@
+### Issue description
+
+Include a description of the issue encountered, and what the expected behavior
+was. Also include any configuration, test suite, or other relevant information
+to reproduce the issue.
```

Comment:
I think this is a very good to starting point. But I feel it is better to have description items as bullet items like below.
What do you think?

```
* Expected behavior and actual behavior
* Steps to reproduce the problem
* Specifications like the version of the project, operating system, or hardware
```

Reply:
I really love this patch idea!

Reply:
I'm fine with bullet points, I'm not tied to the formatting in the current patch at all. My primary motivation was just to make sure we mentioned - - debug, because I always forget about that :)

Reply:
heh, make sense :)

OK, I'm going to approve this, and then, I'll push a patch for the bullet points.